test/check-application

```diff
@@ -148,7 +148,7 @@ class TestApplication(testlib.MachineCase):
 
         # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1008249
         # Fixed in crun 1.20-1
-        self.has_criu = m.image != "debian-stable" and "ubuntu" not in m.image
+        self.has_criu = m.image != "debian-stable" and ("ubuntu" not in m.image or m.image == "ubuntu-stable")
```

Comment:
This is the same no? shouldn't be m.image != ubuntu-stable?

Reply:
This is in `main` already, no need to worry about this bit.